code/__DEFINES/traits/declarations.dm

```diff
@@ -112,6 +112,8 @@ Remember to update _globalvars/traits.dm if you're adding/removing/renaming trai
 //Inherent trait preventing effects of stasis on a mob
 #define TRAIT_NOSTASIS "no_stasis"
 #define TRAIT_MARTIAL_ARTS_IMMUNE "martial_arts_immune" // nobody can use martial arts on this mob
+/// this mob takes reduced damage from falling
+#define TRAIT_CATROBATICS "catrobatics"
```

Comment:
Should be a generalised and sensible trait name, like TRAIT_NO_FALL_DAMAGE

Reply:
Just make a change req

Reply:
```suggestion
#define TRAIT_LIGHT_LANDING "lightlanding"
```